os_stub/spdm_cert_verify_callback_sample/spdm_cert_verify_callback.c

```diff
@@ -0,0 +1,305 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+
+#include <base.h>
+#include "library/memlib.h"
+#include "spdm_cert_verify_callback_internal.h"
+
+/**
+ * tcg-dice-TcbInfo OID: 2.23.133.5.4.1
+ * https://trustedcomputinggroup.org/wp-content/uploads/DICE-Attestation-Architecture-Version-1.1-Revision-18_pub.pdf
+ **/
+uint8_t m_libspdm_tcg_dice_tcbinfo_oid[] = {0x67, 0x81, 0x05, 0x05, 0x04, 0x01};
+
+/*the cert chain must have a or more cert with DiceTcbInfo extension*/
+bool m_libspdm_must_have_dice_tcb_info = true;
+
+/*the reference cert number with DiceTcbinfo in the cert chain*/
+uint8_t m_libspdm_dice_tcb_info_number = 1;
+
+/*reference DiceTcbinfo*/
+
+/*vendor: INTC*/
```

Comment:
If legal, `INTC` -> `DMTF`.

Reply:
We cannot change this. Currently, we are using an internal real DICE cert to do the validation, where the vendor is INTC.